### PR TITLE
Add operating system from vsphere to 'os' field

### PR DESCRIPTION
**Pull request by rfalias**
Add operating system from vsphere to 'os' field

### CODE REVIEW

**Comment by elasticcla** [comment]
Hi @rfalias, we have found your signature in our records, but it seems like you have signed with a different e-mail than the one used in yout Git [commit](https://github.com/elastic/beats/pull/8638.patch). Can you please add both of these e-mails into your Github profile (they can be hidden), so we can match your e-mails to your Github profile?

File: metricbeat/module/vsphere/virtualmachine/virtualmachine.go [hunk 1]
```diff
@@ -141,6 +141,7 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 			event := common.MapStr{
 				"host": vm.Summary.Runtime.Host.Value,
 				"name": vm.Summary.Config.Name,
+				"os": vm.Summary.Config.GuestFullName,
```
Comment (andrewkroh) [inline metricbeat/module/vsphere/virtualmachine/virtualmachine.go]: This file needs to be formatted with `gofmt`.

**Comment by andrewkroh** [comment]
jenkins, test this

**Comment by rfalias** [comment]
Anything need to be done to merge?

**Comment by jsoriano** [comment]
@rfalias sorry for keeping it so long. Could you update the branch again with master and run `make update`? For the rest I think this is good to go.

**Comment by jsoriano** [comment]
@rfalias there are some leftovers from the merge, could you check it? Tell us if you need any help with this.

**Comment by jsoriano** [comment]
I have opened https://github.com/elastic/beats/pull/12391 to continue with this and resolve the merge conflicts so we can merge it.
I am closing this, @rfalias feel free to reopen it if you can continue with this.

**Pull request closed without merging**